src/cloudant/account.py

```diff
@@ -221,14 +201,14 @@ def delete_database(self, dbname):
 
     def db_updates(self, since=None, continuous=True):
         """
-        _db_updates_
-
-        Implement streaming from _db_updates feed. Yields information about
-          databases that have been updated.
+        Streams data from _db_updates feed. Yields information about
+        databases that have been updated.
 
-        :param str since: Start from this sequence
-        :param boolean continuous: Stream results
+        :param str since: Starts update streaming from this sequence
```

Comment:
Adding wording for sequence identifier.  Done in b19046696b22d6b3fb89e04d7d4b7a86ca7d9c41.  This was done throughout the library including db_updates, db_changes and the Feed class.